packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -9,12 +9,14 @@ import {
   EngineEventType,
   getTraceParent,
   getTracingConfig,
+  IsolationLevel,
   LibraryEngine,
   Options,
   runInChildSpan,
   SpanOptions,
   TracingConfig,
 } from '@prisma/engine-core'
+import { BatchTransactionOptions } from '@prisma/engine-core/dist/common/Engine'
```

Comment:
Can you export it via an entry point instead?

Reply:
Oops, auto-import from VS Code misfire.